sopel/irc/__init__.py

```diff
@@ -36,7 +36,7 @@
 from .utils import CapReq, safe
 
 
-__all__ = ['abstract_backends', 'backends', 'utils']
+__all__ = ['abstract_backends', 'backends', 'modes', 'utils']
```

Comment:
To be accessible from autodoc!

Reply:
(or at least I think that's for that? I honestly can't remember)

Reply:
Resurrecting this thread because I noticed one of the force-pushes reverted this. Seems like an accidental change. Was it?

Reply:
100% honest: I've no idea, last comment is like at least a few years old now. 😁 

No but seriously, I've no idea at this point.

Reply:
Reading the list of modified files, I think I went for the safer approach: I didn't modify `sopel/irc/__init__.py` so I don't have to think about adding other sub-modules. If that must be done, then it can be in a future PR.